src/dotnet/APIView/APIViewWeb/TagHelpers/AvatarTagHelper.cs

```diff
@@ -11,8 +11,6 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
         {
             string url = "https://github.com/" + Username + ".png?size=28";
```

Comment:
Don't you need a larger size for the profile page? Maybe add a use a size attribute?

Reply:
@chidozieononiwu Resolved. Interesting to note that if the user has a default GitHub avatar, then the size attribute does nothing (it always gets the max size avatar) which is why none of us picked it up (we all have default profile images).